### PR TITLE
Popup window modal PR

### PR DESCRIPTION
**Pull request by Kossi-stack**
**This project is a pair programming and we did a pop-up window with javascript .**

These are the following requirements for the project.

 - When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
 - When the user clicks (or taps) the close (X) button, the popup disappears.

### CODE REVIEW

File: popup.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+      companyName: 'Uber',
+      job: 'Lead Developer',
+      year: 2018,
+      decription:
+        'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. ',
+      lang: ['html', 'Ruby on rails', 'css', 'javaScript'],
+    },
+  };
+  
+  // Popup menu variables////////////////////
```
Comment (carlos-ssh) [inline popup.js]: [OPTIONAL]
**Keep your project professional:**
- Remove commented lines.

File: style.css [hunk 2]
```diff
@@ -61,6 +61,17 @@ body {
   transition: 500ms ease-in;
 }
```
Comment (carlos-ssh) [inline style.css:63]: - **Kindly fix the style of your project isn't centered.**
Screen Shot 2021-08-04 at 17 17 58

- Remove the exceeded  "X".
Screen Shot 2021-08-04 at 17 20 04

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -9,16 +9,75 @@
...
   <ul >
     <li class="menuList"><a href="#portfolio">Portfolio</a></li>
     <li class="menuList"><a href="#about">About</a></li>
     <li class="menuList"><a href="#contact">Contact</a></li>
   </ul>
   <img src="assets/Icon.png" alt="Close-Menu" class="close" id="close-menu">
-  <div class="mobile-1"></div>
-  <div class="mobile-2"></div>
+  <!--<div class="mobile-1"></div>
+  <div class="mobile-2"></div>-->
```
Comment (bellom) [inline index.html]: It’s better to delete commented code. If you need to review a previous version of your code, use git history.

I trust you can do this before merging.

**Review by bellom — APPROVED** [review]
## Hi Team 👏  

**Great work making all the necessary corrections from the previous review.**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**